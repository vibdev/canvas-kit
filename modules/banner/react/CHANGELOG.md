# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.1.25](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.24...@workday/canvas-kit-react-banner@0.1.25) (2019-06-20)


### Bug Fixes

* broad stroke accessibility fixes ([9803412](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/commits/9803412))





## [0.1.24](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.23...@workday/canvas-kit-react-banner@0.1.24) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.23](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.22...@workday/canvas-kit-react-banner@0.1.23) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.22](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.21...@workday/canvas-kit-react-banner@0.1.22) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.21](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.20...@workday/canvas-kit-react-banner@0.1.21) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.20](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.19...@workday/canvas-kit-react-banner@0.1.20) (2019-06-14)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.19](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.18...@workday/canvas-kit-react-banner@0.1.19) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.18](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.17...@workday/canvas-kit-react-banner@0.1.18) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.17](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.16...@workday/canvas-kit-react-banner@0.1.17) (2019-06-10)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.16](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.15...@workday/canvas-kit-react-banner@0.1.16) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.15](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.14...@workday/canvas-kit-react-banner@0.1.15) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.14](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.13...@workday/canvas-kit-react-banner@0.1.14) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.13](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.12...@workday/canvas-kit-react-banner@0.1.13) (2019-05-31)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.12](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.11...@workday/canvas-kit-react-banner@0.1.12) (2019-05-24)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.11](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.10...@workday/canvas-kit-react-banner@0.1.11) (2019-05-22)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.10](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.9...@workday/canvas-kit-react-banner@0.1.10) (2019-05-21)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.9](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.8...@workday/canvas-kit-react-banner@0.1.9) (2019-05-15)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.8](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.7...@workday/canvas-kit-react-banner@0.1.8) (2019-05-14)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.7](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.6...@workday/canvas-kit-react-banner@0.1.7) (2019-05-10)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.5...@workday/canvas-kit-react-banner@0.1.6) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.5](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.4...@workday/canvas-kit-react-banner@0.1.5) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.4](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.3...@workday/canvas-kit-react-banner@0.1.4) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.2...@workday/canvas-kit-react-banner@0.1.3) (2019-05-07)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.1...@workday/canvas-kit-react-banner@0.1.2) (2019-05-06)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## 0.1.1 (2019-04-30)

**Note:** Version bump only for package @workday/canvas-kit-react-banner
