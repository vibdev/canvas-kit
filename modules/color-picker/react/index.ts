import ColorInput from './lib/ColorInput';
import ColorPreview from './lib/ColorPreview';

export default ColorInput;
export {ColorInput, ColorPreview};
export * from './lib/ColorInput';
export * from './lib/ColorPreview';
