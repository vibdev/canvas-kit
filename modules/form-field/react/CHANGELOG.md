# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.4.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.4.1...@workday/canvas-kit-react-form-field@0.4.2) (2019-07-15)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.4.1](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.4.0...@workday/canvas-kit-react-form-field@0.4.1) (2019-06-20)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





# [0.4.0](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.14...@workday/canvas-kit-react-form-field@0.4.0) (2019-06-19)


### Features

* **Select:** fix from feedback and new folder structure ([cc24691](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/cc24691))
* **Select:** move stories to form field and add to packages ([564d33d](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/564d33d))
* **Select:** resolve conflicts ([48f7d17](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/48f7d17))





## [0.3.14](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.13...@workday/canvas-kit-react-form-field@0.3.14) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.13](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.12...@workday/canvas-kit-react-form-field@0.3.13) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.12](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.11...@workday/canvas-kit-react-form-field@0.3.12) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.11](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.10...@workday/canvas-kit-react-form-field@0.3.11) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.10](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.9...@workday/canvas-kit-react-form-field@0.3.10) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.9](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.8...@workday/canvas-kit-react-form-field@0.3.9) (2019-06-14)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.8](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.7...@workday/canvas-kit-react-form-field@0.3.8) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.7](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.6...@workday/canvas-kit-react-form-field@0.3.7) (2019-06-10)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.5...@workday/canvas-kit-react-form-field@0.3.6) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.5](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.4...@workday/canvas-kit-react-form-field@0.3.5) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.4](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.3...@workday/canvas-kit-react-form-field@0.3.4) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.2...@workday/canvas-kit-react-form-field@0.3.3) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.1...@workday/canvas-kit-react-form-field@0.3.2) (2019-05-31)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.3.1](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.3.0...@workday/canvas-kit-react-form-field@0.3.1) (2019-05-24)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





# [0.3.0](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.2.3...@workday/canvas-kit-react-form-field@0.3.0) (2019-05-23)


### Features

* **ColorPreview:** creating component ([368e189](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/368e189))
* **ColorPreview:** update from feedback ([9ced410](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/9ced410))





## [0.2.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.2.2...@workday/canvas-kit-react-form-field@0.2.3) (2019-05-22)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.2.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.2.1...@workday/canvas-kit-react-form-field@0.2.2) (2019-05-22)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.2.1](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.2.0...@workday/canvas-kit-react-form-field@0.2.1) (2019-05-21)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





# [0.2.0](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.7...@workday/canvas-kit-react-form-field@0.2.0) (2019-05-17)


### Features

* **ColorInput:** exclude story from tsconfig ([4d1b875](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/4d1b875))
* **ColorInput:** use TextInput ([b292d10](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/b292d10))





## [0.1.7](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.6...@workday/canvas-kit-react-form-field@0.1.7) (2019-05-15)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.1.6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.5...@workday/canvas-kit-react-form-field@0.1.6) (2019-05-14)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.1.5](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.4...@workday/canvas-kit-react-form-field@0.1.5) (2019-05-10)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.1.4](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.3...@workday/canvas-kit-react-form-field@0.1.4) (2019-05-09)


### Bug Fixes

* **form-field:** Fix type error in test ([69aed78](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/69aed78))





## [0.1.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.2...@workday/canvas-kit-react-form-field@0.1.3) (2019-05-09)


### Bug Fixes

* **common:** move storybook utils to outside folder ([1f661b6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/1f661b6))





## [0.1.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.1...@workday/canvas-kit-react-form-field@0.1.2) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





## [0.1.1](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/compare/@workday/canvas-kit-react-form-field@0.1.0...@workday/canvas-kit-react-form-field@0.1.1) (2019-05-07)

**Note:** Version bump only for package @workday/canvas-kit-react-form-field





# 0.1.0 (2019-05-06)


### Bug Fixes

* **form-field:** apply uniform height to input elements ([76113ca](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/76113ca))
* **form-field:** Clean up label interface ([86f13a8](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/86f13a8))
* **form-field:** Show errors in the right places ([3544cea](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/3544cea))


### Features

* **common:** Add checkbox support to ControlledComponentWrapper ([54c78fb](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/54c78fb))
* **form-field:** Add aria-invalid if error ([b353aa9](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/b353aa9))
* **form-field:** add error prop to children ([e12d1f6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/e12d1f6))
* **form-field:** add grow prop to children ([cd94681](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/cd94681))
* **form-field:** Add support for aria-describedby ([67a545e](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/67a545e))
* **form-field:** Add support for htmlFor ([ebaf4c1](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/ebaf4c1))
* **form-field:** Add text area to form field stories ([087ed43](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/087ed43))
* **form-field:** Use fieldset and legend for RadioGroups ([48755eb](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-input-common/commits/48755eb))
