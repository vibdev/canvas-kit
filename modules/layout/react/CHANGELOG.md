# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.1.17](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.16...@workday/canvas-kit-react-layout@0.1.17) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.16](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.15...@workday/canvas-kit-react-layout@0.1.16) (2019-06-14)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.15](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.14...@workday/canvas-kit-react-layout@0.1.15) (2019-06-10)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.14](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.13...@workday/canvas-kit-react-layout@0.1.14) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.13](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.12...@workday/canvas-kit-react-layout@0.1.13) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.12](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.11...@workday/canvas-kit-react-layout@0.1.12) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.11](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.10...@workday/canvas-kit-react-layout@0.1.11) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.10](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.9...@workday/canvas-kit-react-layout@0.1.10) (2019-05-31)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.9](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.8...@workday/canvas-kit-react-layout@0.1.9) (2019-05-24)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.8](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.7...@workday/canvas-kit-react-layout@0.1.8) (2019-05-22)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.7](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.6...@workday/canvas-kit-react-layout@0.1.7) (2019-05-21)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.5...@workday/canvas-kit-react-layout@0.1.6) (2019-05-15)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.5](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.4...@workday/canvas-kit-react-layout@0.1.5) (2019-05-14)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.4](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.3...@workday/canvas-kit-react-layout@0.1.4) (2019-05-10)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.2...@workday/canvas-kit-react-layout@0.1.3) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.1...@workday/canvas-kit-react-layout@0.1.2) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





## [0.1.1](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/compare/@workday/canvas-kit-react-layout@0.1.0...@workday/canvas-kit-react-layout@0.1.1) (2019-05-07)

**Note:** Version bump only for package @workday/canvas-kit-react-layout





# 0.1.0 (2019-04-10)


### Features

* **layout:** added fixed width prop for columns ([fe4a860](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/fe4a860))
* **layouts:** added canvas spacing to layouts ([607ed59](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/607ed59))
* **layouts:** added spacing feature ([f388d0d](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/f388d0d))
* **layouts:** content -> capWidth ([93efa87](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/93efa87))
* **layouts:** fixed type errors ([463447a](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/463447a))
* **layouts:** initial commit for layouts component ([ae29eae](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/ae29eae))
* **layouts:** limited columns to 1 through 12 ([3cbacfb](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/3cbacfb))
* **layouts:** made column children optional ([26831f3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/26831f3))
* **layouts:** removed padding from first and last element on column ([9a3e9d3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/9a3e9d3))
* **layouts:** updated readme ([e10218d](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/e10218d))
* **layouts:** updated readme ([8fdc4ce](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-layout/commits/8fdc4ce))
