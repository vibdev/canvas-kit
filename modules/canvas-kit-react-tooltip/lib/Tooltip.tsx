import * as React from 'react';
import styled from 'react-emotion';
import {colors, spacing} from '@workday/canvas-kit-react-core';
import {TransformOrigin} from '@workday/canvas-kit-react-common';
import {keyframes} from 'emotion';

export interface TooltipProps {
  transformOrigin: TransformOrigin;
}

const tooltipAnimation = keyframes`
  0% {
    opacity: 0;
    transform: scale(0.8);
  }
  100% {
    opacity: 1;
    transform: scale(1);
  }
  `;

const TooltipContainer = styled('div')<TooltipProps>(
  {
    display: 'inline-flex',
    borderRadius: spacing.xxxs,
    padding: spacing.xxs,
    backgroundColor: 'rgba(0,0,0,.85)',
    animation: tooltipAnimation,
    animationDuration: '150ms',
    animationTimingFunction: 'ease-out',
    color: colors.frenchVanilla100,
    fontSize: 13,
    margin: spacing.xxxs,
    a: {
      color: colors.frenchVanilla100,
      textDecoration: 'underline',
    },
  },
  ({transformOrigin}) => ({
    transformOrigin: transformOrigin
      ? `${transformOrigin.vertical} ${transformOrigin.horizontal}`
      : 'top center',
  })
);

export default class Tooltip extends React.Component<TooltipProps, {}> {
  static defaultProps = {
    transformOrigin: {
      horizontal: 'center',
      vertical: 'bottom',
    },
  };

  public render() {
    return (
      <TooltipContainer {...this.props} role="tooltip">
        {this.props.children}
      </TooltipContainer>
    );
  }
}
