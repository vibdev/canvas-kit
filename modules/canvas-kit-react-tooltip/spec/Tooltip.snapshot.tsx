import * as renderer from 'react-test-renderer';
import Tooltip from '../lib/Tooltip';
import * as React from 'react';
jest.mock('react-dom');
describe('Tooltip Snapshots', () => {
  test('renders a tooltip with a child element', () => {
    const container = renderer.create(
      <div>
        <Tooltip>
          <div>I'm being hovered</div>
        </Tooltip>
      </div>
    );
    const tree = container.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders a tooltip with a different transformOrigin', () => {
    const container = renderer.create(
      <div>
        <Tooltip transformOrigin={{horizontal: 'center', vertical: 'top'}}>
          <div>Delete</div>
        </Tooltip>
      </div>
    );
    const tree = container.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
