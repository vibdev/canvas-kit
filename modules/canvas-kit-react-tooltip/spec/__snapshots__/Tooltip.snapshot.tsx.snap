// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip Snapshots renders a tooltip with a child element 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  border-radius: 4px;
  padding: 8px;
  background-color: rgba(0,0,0,.85);
  -webkit-animation: animation-fmkrp5;
  animation: animation-fmkrp5;
  -webkit-animation-duration: 150ms;
  animation-duration: 150ms;
  -webkit-animation-timing-function: ease-out;
  animation-timing-function: ease-out;
  color: #ffffff;
  font-size: 13px;
  margin: 4px;
  -webkit-transform-origin: top center;
  -ms-transform-origin: top center;
  transform-origin: top center;
}

.emotion-0 a {
  color: #ffffff;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<div>
  <div
    className="emotion-0 emotion-1"
    role="tooltip"
  >
    <div>
      I'm being hovered
    </div>
  </div>
</div>
`;

exports[`Tooltip Snapshots renders a tooltip with a different transformOrigin 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  border-radius: 4px;
  padding: 8px;
  background-color: rgba(0,0,0,.85);
  -webkit-animation: animation-fmkrp5;
  animation: animation-fmkrp5;
  -webkit-animation-duration: 150ms;
  animation-duration: 150ms;
  -webkit-animation-timing-function: ease-out;
  animation-timing-function: ease-out;
  color: #ffffff;
  font-size: 13px;
  margin: 4px;
  -webkit-transform-origin: top center;
  -ms-transform-origin: top center;
  transform-origin: top center;
}

.emotion-0 a {
  color: #ffffff;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<div>
  <div
    className="emotion-0 emotion-1"
    role="tooltip"
  >
    <div>
      Delete
    </div>
  </div>
</div>
`;
